torch/nn/utils/rnn.py

```diff
@@ -215,7 +215,8 @@ def pack_padded_sequence(input, lengths, batch_first=False, enforce_sorted=True)
 
     Arguments:
         input (Tensor): padded batch of variable length sequences.
-        lengths (Tensor): list of sequences lengths of each batch element.
+        lengths (Tensor or list(int)): list of sequences lengths of each batch
```

Comment:
"sequences" -> "sequence"

"if provided as a Tensor" -> "if provided as a tensor" (tensor is lowercase when used in descriptions in the docs)